semanticizest/parse_wikidump/__main__.py

```diff
@@ -52,53 +83,45 @@ def die(msg):
     sys.exit(1)
 
 
-def main(args):
-    args = docopt(__doc__, args)
-
-    if args["--download"]:
-        wikidump = args["--download"] + ".xml.bz2"
-    else:
-        wikidump = args['<dump>']
-
-    model_fname = args['<model-filename>']
-    ngram = args['--ngram']
-    if ngram == "None":
-        ngram = None
-    else:
-        ngram = int(ngram)
-
-    logger.info("Creating database at %r" % model_fname)
-    try:
-        db = sqlite3.connect(model_fname)
-    except sqlite3.OperationalError as e:
-        if 'unable to open' in str(e):
-            # This exception doesn't store the path.
-            die("%s: %r" % (e, model_fname))
-        else:
-            raise
-    with open(createtables_path()) as f:
-        create = f.read()
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(prog="semanticizer.parse_wikidump", description="Semanticizest Wiki parser")
+    parser.add_argument('snapshot', 
+                        help='Local Wikipedia snapshot to use.')
+    parser.add_argument('model', 
+                        help='File to store the model.')
+    parser.add_argument('--download', dest='download', action="store_true",
+                        help='Download snapshot if it does not exist as snapshot.xml.bz2. The corpus file name should match that of snapshot.')
+    parser.add_argument('-N', '--ngram', dest='ngram', default=7, type=int,
+                        help='Maximum order of ngrams, set to None to disable [default: 7].')
+    args = parser.parse_args()
```

Comment:
I think @IsaacHaze will be very unhappy when he sees his darling `docopt` replaced by `ArgumentParser`...